app/routes/articles/components/ArticleEditor.tsx

```diff
@@ -179,6 +180,15 @@ const ArticleEditor = () => {
               type="checkbox"
               component={CheckBox.Field}
             />
+            {values.pinned && (
+              <Card style={{ marginTop: 0 }} severity="warning">
+                <span>
+                  OBS! Du må ha godkjenning fra ledelsen for å feste til
```

Comment:
Could the "obs" be in a `Card.Header` so that it matches other cards better?😁

Reply:
Done!